pom.xml

```diff
@@ -208,11 +201,6 @@
                             <configuration>
                                 <shadedClassifierName>uber</shadedClassifierName>
                                 <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
```

Comment:
-1: Remove this promoteTransitiveDependencies, as now unnecessary

This was introduced to have "algoliasearch-apache-uber" list "slf4j-api" as a dependency in its reduced pom.